CHANGES.md

```diff
@@ -1,3 +1,20 @@
+## edge-19.10.4
+
+This edge release adds dashboard UX enhancements, Open Census tracing
```

Comment:
s/Open Census/OpenCensus

Reply:
Also I wonder if we should omit mentioning this for now, as I don't think there is a direct way for users to use it until https://github.com/linkerd/linkerd2/pull/3539 is merged.